Description: Do auto-shutdown when idle also with active web interface.
 Turning of the web interface removes important functionality of CUPS.
Author: Till Kamppeter <till.kamppeter@gmail.com>
Bug: https://cups.org/str.php?L4755
Last-Update: 2016-01-12
--- a/scheduler/main.c
+++ b/scheduler/main.c
@@ -790,9 +790,6 @@
 
     if (timeout == 86400 && OnDemand && IdleExitTimeout &&
         !cupsArrayCount(ActiveJobs) &&
-#  ifdef HAVE_SYSTEMD
-        !WebInterface &&
-#  endif /* HAVE_SYSTEMD */
 	(!Browsing || !BrowseLocalProtocols || !cupsArrayCount(Printers)))
     {
       timeout		= IdleExitTimeout;
